src/Infrastructure.VS/Infrastructure.VS.csproj

```diff
@@ -18,7 +18,7 @@
 
   <ItemGroup Label="Package references">
     <!-- The VS.Setup.Config.Interop package is not in the v17 SDK meta-package, but it is in the v15.0.1 meta-package -->
-    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.16.30" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="3.3.2180" PrivateAssets="all" />
```

Comment:
It looks like the [newer SDK package](https://www.nuget.org/packages/Microsoft.VisualStudio.SDK#dependencies-body-tab) does include a reference to `Microsoft.VisualStudio.Setup.Configuration.Interop`, so this direct reference should no longer be necessary.